recipes/d3rlpy/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "d3rlpy" %}
+{% set version = "0.32" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 808ff3094b0e59ff373213a02034158f798d1a5c378ab5de222629148267676b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
Does this links to numpy? If so please read https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
@ocefpaf Thank you!! I've fix this part!

Reply:
BTW, [try grayskull](https://github.com/conda-incubator/grayskull). It should implement that automatically for you and ensure you have the latest recipe format.